content/2017-09-19_pycon-cz-conference-wrap-up-summary.md

```diff
@@ -0,0 +1,199 @@
+
+- **Stream Thursday**
+    - Main Room - 962 views, 13172 minutes watch time, 20 likes
+    - Theatre Room - 563 views, 4662 minutes watch time, 10 likes
+- **Stream Friday**
+    - Main Room - 794 views, 7717 minutes watch time, 11 likes
+    - Theatre Room - 359 views, 3780 minutes watch time, 3 likes
+- **Videos with the most views**:
+    - [Armin Ronacher: A Python and Rust love story](https://cz.pycon.org/2017/speakers/detail/talk/30/)
+    - [Naomi Ceder: What lies ahead? Python’s future strengths and challenges](https://cz.pycon.org/2017/speakers/detail/talk/34/)
+    - [Karla Fejfarová & Petr Šimeček: Python alternatives to R/Shiny](https://cz.pycon.org/2017/speakers/detail/talk/8/)
+
+## Socially Responsible Catering
+
+Catering was provided by the coffee shop & bakery [Slunce](http://www.slunce.info/index.php?option=com_content&view=article&id=178&Itemid=91), which employs people with disabilities. The activities of Slunce are formally backed by the [Sun for Everyone Endowment Fund](http://www.slunce.info/index.php?option=com_content&view=article&id=576&Itemid=78) ([Czech](http://slunce.info/)).
+
+## Diversity
+
+We've put effort into making the conference as welcoming as possible. We had [Code of Conduct](https://cz.pycon.org/2017/about/code/) in place and we were reaching out to specific communities, which are usually underrepresented in technical conferences.
+
+The women/men ratio of the organizing team was roughly 50:50. After moving the conference from Brno to Prague, the Prague PyLadies beginners course alumni [Bára Drbohlavová](https://twitter.com/BaraDrb) volunteered to become the chair of the conference. The [PyLadies of Prague, Brno and Ostrava](http://pyladies.cz/) were invited to attend with a discount equal to student/teacher ticket fare.
+
+[Zuzka Válková](https://github.com/zuzejk), long time Prague PyLadies beginners course organizer, joined our team to hold the [Django Girls](https://djangogirls.org/) workshop together with the conference. All Django Girls participants had free entrance to the conference.
+
+Vojtěch Polášek, author of the [Blind attendee about PyCon CZ 2016](http://blog.python.cz/blind-attendee-about-pycon-cz-2016) blog post, came to the conference also this year.
+
+### Diversity Stats
+
+- More than 30% of the talks were not given by men. The selection process did not take gender into consideration.
+- More than ⅕ of the audience were women.
+
+From the detailed analysis of the data we have it seems that women tend to buy cheaper tickets than men, probably mostly because in the context of the Czech Python community they're often also beginners. The data shows that the ticket price is important for achieving inclusivity. Without the discounts aimed towards women (Django Girls, PyLadies) and without the Financial Aid we would see much less women attending the conference.
```

Comment:
often beginners as well.
we would see much fewer women

Reply:
Mozna zminit neco v tom smyslu, ze pro Diversity je pro nas dulezity nejen men/women, ale i pro/beginners? 

Zarven bych mozna misto "they're often also beginners" rekl, ze "nejsou casto day job py developers", popr ze "jsou hobbysts"?